src/Bokeh.jl

```diff
@@ -1,12 +1,15 @@
+VERSION >= v"0.4.0-dev+6641" && __precompile__()
```

Comment:
an alternative would be feature testing instead of version checks: `isdefined(Base, :__precompile__)`